health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/SelectingBoundaryComponent.js

```
@@ -444,163 +444,169 @@ const SelectingBoundaryComponent = ({
+              // Find the count of boundaryType in boundaryOptions
+              const optionsCount = Object.entries(boundaryOptions)
+                .filter(([key]) => key.startsWith(boundary.boundaryType))
+                .flatMap(([key, value]) => Object.entries(value || {})).length;
+
+              // Disable dropdown if counts match
+              const isDisabled = frozenCount === optionsCount;
+
+              return boundary?.parentBoundaryType == null ? (
+                <LabelFieldPair style={{ alignItems: "flex-start", paddingRight: "30%" }}>
+                  <CardLabel className={"boundary-selection-label"}>
+                    {t((hierarchyType + "_" + boundary?.boundaryType).toUpperCase())}
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)